examples/gallery/lines/contours.py

```diff
@@ -0,0 +1,25 @@
+"""
+Contours
+--------
+The :meth:`pygmt.Figure.contour` method can plot contour lines from a table of points.
+"""
+
+
+import numpy as np
+import pygmt
+
+X, Y = np.meshgrid(np.linspace(-10, 10, 50), np.linspace(-10, 10, 50))
+Z = X**2 + Y**2
```

Comment:
```suggestion
Z = X ** 2 + Y ** 2
```
This is just a formatting suggestion that was caught by our linter.